URBNSwiftyConvenience/Classes/Border.swift

```diff
@@ -8,19 +8,35 @@
 
 import UIKit
 
+/// Reuseable value type border view model that can be used to initalize a new border instance
+public struct BorderStyle {
+    
+    public let color: UIColor
+    public let pixelWidth: CGFloat
+    public let insets: UIEdgeInsets
+    
+    public init(color: UIColor, pixelWidth: CGFloat = 1, insets: UIEdgeInsets = .zero) {
+        self.color = color
+        self.pixelWidth = pixelWidth
+        self.insets = insets
+    }
+}
+
 /// Represents a border with a with a width, a color, and insets
-public final class Border: UIView {
+final class Border: UIView {
```

Comment:
Why not make the whole class `fileprivate` then?

Reply:
Border is used in `UIView+Additions` so it can't be fileprivate.